Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -240,6 +242,31 @@ public final class PackageInfoMapper: PackageInfoMapping {
                     }
             }
 
+        var macroTargetsNames = Set(packageInfos.values.flatMap { $0.targets.filter { $0.type == .macro }.map(\.name) })
```

Comment:
nit: let's try to be specific with the name, this also includes their dependencies, right?
```suggestion
        var macroTargetsAndDescendants = Set(packageInfos.values.flatMap { $0.targets.filter { $0.type == .macro }.map(\.name) })
```